homeassistant/components/fan/__init__.py

```diff
@@ -125,7 +124,7 @@ def is_on(hass, entity_id: str) -> bool:
     return state.state == STATE_ON
 
 
-async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+async def async_setup(hass, config: dict):
```

Comment:
Please keep this as before.

Reply:
Huh. I wonder how I managed to mess this up on so many levels 😕 I think I've checked out the file from dev when I reverted the changes I made.

I'll fix this tomorrow.